### ISSUE TITLE
handaling of empty string in json to xml  to json conversion

### ISSUE THREAD

**Issue by rajeevaec** [beckchr/staxon#32]
I an using staxon to convert json to xml and then same xml to json.
that is json <-->xml<-->json, 
while doing so when I am giving following json
{
"emptyKey" : "",
"nullKey" :null
} 
which is getting converted into this xml with virtual root
<?xml version="1.0"?>
<root>
<emptyKey></emptyKey>
<nullKey></nullKey>
</root>
now when I convert this xml back to JSON I am getting json
{
  "emptyKey" : null,
  "nullKey" : null
}

basically empty string is converted into null, 
Is there any way to handle empty string properly.

**Comment by beckchr** [comment]
There is currently no way to handle this, i.e. there is no `null` in XML. There is a pull request to support the `xsi:nil` attribute, see https://github.com/beckchr/staxon/pull/27. However, this needs some more investigation.
